components/components/header.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, SIZE } from "baseui/button";
+import * as React from "react";
+import styles from "./styles.module.css";
+
+
+export const Header = function () {
```

Comment:
Keep you file name as Header

Reply:
Added NavigationBar and Header to Component.tsx file